exercise02/exercises09.js

```diff
@@ -0,0 +1,4 @@
+function calculateMatrixSum(matrix) {
+  const flatternMatrix = matrix.flat();
```

Comment:
`flatternMatrix` sounds like a function, I know finding names sometimes can be challenging but we should alway give it time to avoid future confusions